server/app.ts

```diff
@@ -327,355 +296,6 @@ apiRouter.route("/user/login").post(postParser, async (request, response) => {
-	}
-	let tagNames: string[] = tag.toLowerCase().split(/, */);
-	let tags: ITags = {};
-	for (let i = 0; i < tagNames.length; i++) {
-		tags[tagNames[i]] = {checked_in: false};
-	}
-	tag = tag.trim().toLowerCase();
-	nameHeader = nameHeader.trim();
-	let emailHeaders: string[] = emailHeadersRaw.split(",").map((header) => { return header.trim(); });
-
-	parser.on("readable", () => {
-		let record: any;
-		while (record = parser.read()) {
-			if (!headerParsed) {
-				// Header row
-				for (let i = 0; i < record.length; i++) {
-					let label: string = record[i];
-
-					if (label.match(new RegExp(`^${reEscape(nameHeader)}$`, "i"))) {
-						nameIndex = i;
-					}
-					for (let emailHeader of emailHeaders) {
-						if (label.match(new RegExp(`^${reEscape(emailHeader)}$`, "i"))) {
-							emailIndexes.push(i);
-						}
-					}
-				}
-				headerParsed = true;
-			}
-			else {
-				// Content rows
-				if (nameIndex === null || emailIndexes.length === 0) {
-					throw new Error("Invalid header names");
-				}
-				// Capitalize names
-				let name: string = record[nameIndex] || "";
-				name = name.split(" ").map(s => {
-					return s.charAt(0).toUpperCase() + s.slice(1)
-				}).join(" ");
-
-				let emails: string[] = [];
-				for (let emailIndex of emailIndexes) {
-					if (record[emailIndex])
-						emails.push(record[emailIndex]);
-				}
-
-				if (!name || emails.length === 0) {
-					console.warn("Skipping due to missing name and/or emails", record);
-					continue;
-				}
-
-				attendeeData.push({
-					name: name,
-					emails: emails,
-					id: crypto.randomBytes(16).toString("hex"),
-					tags: tags
-				});
-			}
-		}
-	});
-	let hasErrored: boolean = false;
-	parser.on("error", (err: Error) => {
-		hasErrored = true;
-		if (err.message !== "Invalid header names") {
-			console.error(err);
-		}
-		response.status(415).json({
-			"error": "Invalid header names or CSV"
-		});
-	});
-	parser.on("finish", async () => {
-		if (hasErrored)
-			return;
-		if (attendeeData.length < 1) {
-			response.status(415).json({
-				"error": "No entries to import"
-			});
-			return;
-		}
-		let attendees: IAttendeeMongoose[] = attendeeData.map((attendee) => {
-			return new Attendee(attendee);
-		});
-		try {
-			await Attendee.insertMany(attendees);
-			response.status(200).json({
-				"success": true
-			});
-		}
-		catch (err) {
-			if (err.code === 11000) {
-				response.status(409).json({
-					"error": "Name duplication detected. Please clear the current attendee list before importing this new list."
-				});
-				return;
-			}
-			console.error(err);
-			response.status(500).json({
-				"error": "An error occurred while saving users to the database"
-			});
-		}
-	});
-	if (!request.file) {
-		response.status(400).json({
-			"error": "No CSV file to process and import"
-		});
-		return;
-	}
-	fs.createReadStream(request.file.path).pipe(parser);
-});
-
-apiRouter.route("/data/export").get(authenticateWithReject, async (request, response) => {
```

Comment:
Is CSV data export something worth keeping around? Related to #48

Reply:
Yeah, this is a good point. Let's keep CSV export, but I do think we should get around to #48 at some point, since the CSV export exports data in checkin's mongo collection, which is only a subset of all the data that checkin GraphQL can provide.